src/VitessPdo/PDO/ParamProcessor.php

```diff
@@ -108,6 +112,35 @@ private function getHandler($type)
     }
 
     /**
+     * @param $value
+     *
+     * @return int
+     */
+    private function determineValueType($value){
```

Comment:
does this ever return something else then PARAM_NULL ? it is only called with null value as the $value parameter